aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -28,21 +28,15 @@ struct Vec256 {
       values[i] = val;
     }
   }
-  void load(const void* ptr) {
-    std::memcpy(values, ptr, 32);
-  };
-  void load_partial(const void* ptr, int count) {
+  void load(const void* ptr, int count = size) {
```

Comment:
nit: I'd use `uint64_t` or `size_t` for sizes. That's generally going to incur fewer type conversions inside the kernels, and that's good. Currently it will need to be promoted to a larger type inside this function because you multiply it with `sizeof`, which is a `size_t`

Reply:
Using int64_t because of Sam's comment below.